src/v3/bandwidthRtc.ts

```diff
@@ -421,16 +434,20 @@ class BandwidthRtc {
 
   private setupNewPeerConnection(peerConnection: RTCPeerConnection, onPeerClosed: CallableFunction): void {
     peerConnection.onconnectionstatechange = (event: Event) => {
-      const pc = event.target as RTCPeerConnection;
-      logger.debug("onconnectionstatechange", pc.connectionState, pc);
-      const connectionState = pc.connectionState;
-      if (connectionState === "disconnected") {
-        logger.warn("Peer disconnected, connection may be reestablished");
-      }
-      if (connectionState === "failed" || connectionState === "closed") {
-        logger.warn("Connection lost, refresh to retry");
-        // TODO: make automatic reconnection work
-        // onPeerClosed();
+      try {
+        const pc = event.target as RTCPeerConnection;
+        logger.debug("onconnectionstatechange", pc.connectionState, pc);
+        const connectionState = pc.connectionState;
+        if (connectionState === "disconnected") {
+          logger.warn("Peer disconnected, connection may be reestablished");
+        }
+        if (connectionState === "failed" || connectionState === "closed") {
+          logger.warn("Connection lost, refresh to retry");
+          // TODO: make automatic reconnection work
```

Comment:
TODO removal

Reply:
I'm going to (briefly) leave this, it should hopefully be TODONE in the next few days.